curriculum/challenges/english/25-front-end-development/lab-lightbox-viewer/66db57ad34c7089b9b41bfd6.md

```diff
@@ -230,6 +230,29 @@ background.dispatchEvent(new Event('click'));
 // Wait for any async operations if needed
 // await new Promise(resolve => setTimeout(resolve, 100)); // Adjust timeout as needed
 
+// Assert that the lightbox is hidden after clicking outside
+assert.strictEqual(getComputedDisplay(lightbox), 'none');
+```
+When your `.lightbox` element is visible and you click the `.lightbox` element, the `.lightbox` elements `display` should be set back to `none`.
```

Comment:
````suggestion
assert.strictEqual(getComputedDisplay(lightbox), 'none');
```

When your `.lightbox` element is visible and you click the `.lightbox` element, the `.lightbox` elements `display` should be set back to `none`.
````
code blocks need to be surrounded by empty lines, I'm surprised the pre-commit hooks allowed you to make the commit